src/XamarinBackgroundKit.iOS/Renderers/MaterialContentViewRenderer.cs

```diff
@@ -122,14 +109,12 @@ protected override void Dispose(bool disposing)
             {
                 if (BackgroundManager != null)
                 {
-                    BackgroundManager.Dispose();
+                    BackgroundManager?.Dispose();
```

Comment:
BackgroundManager should not be null there

Reply:
Yes, sorry!